src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -2644,6 +3005,73 @@ mod tests {
             assert!(test_bed.yellow_pressure() < Pressure::new::<psi>(500.));
         }
 
+        #[test]
+        fn yellow_edp_solenoid_unpowered() {
+            let mut test_bed = test_bed_with()
+                .engines_off()
+                .on_the_ground()
+                .set_cold_dark_inputs()
+                .run_one_tick();
+
+            // Starting eng 2
+            test_bed = test_bed
+                .start_eng2(Ratio::new::<percent>(80.))
+                .run_waiting_for(Duration::from_secs(15));
+
+            assert!(test_bed.is_yellow_pressurised());
+
+            // Stoping EDP
+            test_bed = test_bed
+                .set_yellow_ed_pump(false)
+                .run_waiting_for(Duration::from_secs(15));
+
+            assert!(!test_bed.is_yellow_pressurised());
```

Comment:
I don't know if these are part of the same test, or if you're actually intending to test multiple scenarios.